teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
+| Facility Locator Search - Facility Type | 110 | Hit | Facility type (for example, `health` or `pharmacy`) collected from a facility locator search on this page: www.va.gov/facilities/ |
+| Health Care VAOS - Returned Item Type | 111 | Hit | Returned item type (for example, `upcoming` or `video_home`) from the VAOS flow and collected on this page: www.va.gov/health-care/schedule-view-va-appointments/appointments/  |
+| Health Care VAOS - Returned Number of Items | 112 | Hit | CD is set to an integer matching the number of items of a search on this page:         <br>www.va.gov/health-care/schedule-view-va-appointments/appointments/ |
+| Global - Dependent Debt | 113 | User | CD is set to either `Without Dependent Debt` or `With Dependent Debt` depending on the user's selection on this page: www.va.gov/manage-va-debt |
+| Type Ahead Options Keyword Selected | 114 | Hit | User selects a type ahead option. The values for this CD are endless. |
+| Type Ahead Options List | 115 | Hit | User views type ahead options list |
+| Phased Out Product | 116 | Session | CD value is set to either `type ahead` or `chapter 36` |
+| Type Ahead Option Chosen Position | 117 | Hit | CD value is set to the position of the type ahead option that the user selected (either `1`, `2`, `3`, `4`, `5`, or `(not set)`) |
+| HLR - Same Office Requested | 118 | Hit | CD value is set to either `yes` or `no` based on the user's selection on the 20-0996 form |
+| HLR - Informal Conference Requested | 119 | Hit | CD value is set to either `yes`, `yes-with-rep`, or `no` based on the user's selection on informal conference requested the 20-0996 form |
+| Search Results Type | 120 | Hit | CD value is set to the search result type (`title`, `pdf`, `cta`, etc.) a user selects when completing a search on any search page ( /search/, /find forms/, /yellow-ribbon-participating-schools/, etc.) |
+| Medallia Feedback UUID | 121 | Session | The UUID for Medallia feedback typically found on this page: www.va.gov/facilities/index.html?XXX |
+| Medallia Survey ID | 122 | Hit | Survey ID for the Medallia survey that is being leveraged |
+| Medallia Survey Status | 123 | Hit | The Medallia survey status is set to either `AlwaysOn`, `Intercept`, or `Invite` and is collected when a user begins a survey |
+| Placeholder - CLP Event Name | 124 | Hit | **No data is being collected** |
+| Medallia - Survey Contents | 125 | Hit | String of information collected when a survey is submitted or closed |
+| Education - Facility Code | 126 | Hit | This CD is most commonly set as `(not set)`, however a facility code is captured if a user enters a facility on the page: www.va.gov/education/yellow-ribbon-participating-schools/ |
+| Education - Institution ID | 127 | Hit | This CD is most commonly set as `(not set)`, however an institution ID is captured if a user enters the relevant information on the page: www.va.gov/education/yellow-ribbon-participating-schools/ |
```

Comment:
_There's probably a good amount of cases where there is no institution ID, however we have their developers push this from the backend on search results click_

```suggestion
| Education - Institution ID | 127 | Hit | This CD is most commonly set as `(not set)`, however an institution ID is captured if a user selects a search result option with an institution ID
```